ui_setup.py

```diff
@@ -237,4 +241,7 @@ def runSimWithUI(self):
         self.UIMap = ui_sim.UISim(map_width,map_height,self.sim,self.omsgr,self,self.logger)
 
     def runSimWithoutUI(self):
-        self.sim.runSim(self.ldr.getMainConfigData('no_ui_max_turns'))
```

Comment:
Whats the difference between this line and the new line 244? I can't see a visible difference so I'm wondering if it's a change to the whitespace that might cause a problem.

Reply:
I think it doesn't have a new line character or something at the end? I have tested my changes and have had no issues running it.